package-resources.yml

```diff
@@ -0,0 +1,58 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: Package
+metadata:
+  name: secretgen-controller.carvel.dev.0.0.0
+spec:
+  refName: secretgen-controller.carvel.dev
+  releasedAt: null
+  template:
+    spec:
+      deploy:
+      - kapp: {}
+      fetch:
+      - git: {}
+      template:
+      - ytt:
+          paths:
+          - config/package-bundle/config
+          - config/release
+      - kbld:
+          paths:
+          - config/release
+  valuesSchema:
+    openAPIv3: null
+  version: 0.0.0
+
+---
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  name: secretgen-controller.carvel.dev
+spec:
+  displayName: "secretgen-controller"
+  longDescription: "Provide CRDs to specify what secrets need to be on cluster (generated or not)."
+  shortDescription: "Secret generation and sharing"
+  providerName: carvel.dev
+  maintainers:
+    - name: Carvel team (see www.carvel.dev for team members)
+  categories:
+    - "package management"
+
+---
+apiVersion: packaging.carvel.dev/v1alpha1
```

Comment:
What is the PKGI used for when running the release steps of kctrl? We don't publish it right? Or does this also include a PKGI to our release assets?

Reply:
No we won't publish it and is not part of release artifact

Reply:
so is it just something for kctrl to use? Or is it needed? I haven't dived into the kctrl flow just yet, so my questions are a little naive :)

Reply:
Yeah it is something needed for kctrl